types/artifacts.d.ts

```diff
@@ -326,7 +326,7 @@ declare global {
           files: Record<string, number>;
           unmappedBytes: number;
           totalBytes: number;
-        };
+        } | {error: string};
```

Comment:
bikeshed: usually we keep `error` for some kind of error object. `errorMessage`?